logp/core.go

```diff
@@ -236,6 +236,22 @@ func makeEventLogOutput(cfg Config, enab zapcore.LevelEnabler) (zapcore.Core, er
 	return wrappedCore(core), nil
 }
 
+// WithFileOutput creates a new file output based on cfg and
+// replaces the previous one.
+func WithFileOutput(cfg Config) func(zapcore.Core) zapcore.Core {
```

Comment:
Is it possible to write a test for this?

Reply:
Done on 76d680c